README.md

```diff
@@ -46,12 +46,56 @@ ls target/dspot-*-jar-with-dependencies.jar
 java -cp target/dspot-*-jar-with-dependencies.jar fr.inria.diversify.Main path/To/my.properties
 ```
 
+### Command Line Usage
+```
+Usage: java -jar target/dspot-1.0.0-jar-with-dependencies.jar
+                          [(-p|--path) <path>] [(-a|--amplifiers) Amplifier1:Amplifier2:...:AmplifierN ] [(-i|--iteration) <iteration>] [(-s|--selector) <BranchCoverageTestSelector | PitMutantScoreSelector>] [(-t|--test) test1:test2:...:testN ] [(-o|--output) <output>] [(-m|--mutant) <mutant>] [-e|--example] [-h|--help]
+
+  [(-p|--path) <path>]
+        specify the path to the configuration file of the target project.
+
+  [(-a|--amplifiers) Amplifier1:Amplifier2:...:AmplifierN ]
+        specify the list of the amplifiers to use.
+        value for Amplifier:
+        MethodAdd|MethodRemove|StatementAdderOnAssert|TestDataMutator
+        (default:
+        TestDataMutator:TestMethodCallAdder:TestMethodCallRemover:StatementAdderOnAssert)
+
+  [(-i|--iteration) <iteration>]
+        specify the number of time each amplifiers is applied (default: 3)
+
+  [(-s|--selector) <BranchCoverageTestSelector | PitMutantScoreSelector>]
```

Comment:
rename to "test-criterion'.

[optional] specify the test adequacy criterion to be maximized with amplification

example default: coverage
command line default: PIT